### PR TITLE
plot co2e and energy in one plot with two axis

### PR DESCRIPTION
**Pull request by mirpedrol**
image

### CODE REVIEW

File: plugins/nf-co2footprint/src/resources/CO2FootprintReportTemplate.html [hunk 1]
```diff
@@ -235,41 +235,8 @@ <h3 class="mt-5">Total CO<sub>2</sub>e footprint measures</h3>
...
-        <div id="energyplot"></div>
-      </div>
-      <div class="tab-pane fade" id="pctenergyplot_tabpanel" role="tabpanel">
-        <div id="pctenergyplot"></div>
-      </div>
-    </div>
-  </div>
-
+    <div id="co2eplot"></div>
+    <p>CO2 emission values are depicted on the plot. The proportional values of energy consumption are represented on the right y-axis.</p>
```
Comment (skrakau) [inline plugins/nf-co2footprint/src/resources/CO2FootprintReportTemplate.html:239]: Maybe: 
```suggestion
    <p>The plot shows the distribution of CO<sub>2</sub>e emissions and corresponding energy consumptions for each workflow process. The values that are displayed when hovering over the plot refer to the CO<sub>2</sub>e emissions.</p>
```

**Comment by skrakau** [comment]
Sorry for the late review @mirpedrol !

File: plugins/nf-co2footprint/src/resources/assets/CO2FootprintReportTemplate.js [hunk 2]
```diff
@@ -66,44 +66,43 @@ $(function() {
...
+  },
+  {
+    "dtickrange": [4, null],
+    "value": ".3s"
+  }];
 
-  Plotly.newPlot('co2eplot', co2e_data, { title: 'CO2e emission', yaxis: {title: 'CO2e emission (g)', tickformat: co2e_tickformat, rangemode: 'tozero'} });
-  Plotly.newPlot('energyplot', energy_data, { title: 'Energy consumption', yaxis: {title: 'Energy consumption (Wh)', tickformat: energy_tickformat, rangemode: 'tozero'} });
+  var layout = {
+    title: 'CO2 emission & energy consumption',
```
[the remark this answers is not in the document]
Comment (mirpedrol) [reply plugins/nf-co2footprint/src/resources/assets/CO2FootprintReportTemplate.js:89]:
```suggestion
    title: 'CO<sub>2</sub> emission & energy consumption',
```

**Comment by mirpedrol** [comment]
> Did something change regarding displaying the values? I was wondering why the median is displayed as 16.4 not and not 16.40. Wasn't this like this before?

This is due to a new way I found to determine which number of decimals to use: https://github.com/nextflow-io/nf-co2footprint/pull/87/files#diff-c42fa18c056e4a0988d45f4a58d2604ec610ff4e979f175cdf8243ee3f235890R79-R86
now, instead of using the maximum value of the whole dataset to determine the number of decimal positions, we do it case by case, values of >= 4 will have only 1 position

**Pull request merged**